server/manager/src/main/java/org/apache/accumulo/manager/FateServiceHandler.java

```diff
@@ -505,7 +505,7 @@ public void executeFateOperation(TInfo tinfo, TCredentials c, TFateId opid, TFat
         TableOperation tableOp = TableOperation.DELETE_RANGE;
         validateArgumentCount(arguments, tableOp, 3);
         String tableName =
-            validateName(arguments.get(0), tableOp, EXISTING_TABLE_NAME.and(NOT_METADATA_TABLE));
+            validateName(arguments.get(0), tableOp, EXISTING_TABLE_NAME.and(NOT_BUILTIN_TABLE));
```

Comment:
Maybe reverse the checks, because checking for a built-in table is faster than checking if it exists.